assets/component-menu-drawer.css

```diff
@@ -235,7 +235,7 @@ details[open].menu-opening > .menu-drawer__submenu {
 }
 
 @media screen and (min-width: 750px) {
-  .menu-drawer__utility-links:has(ul:nth-child(2):empty) {
+  .menu-drawer__utility-links:has(ul:empty) {
```

Comment:
That doesn't work properly because locale selectors should be coming up in the drawer on tablet sizes (`750px - 990px`). And with this it doesn't account for it. 

Because we're hiding the wrapper just because of social media when we're not checking on the locales being enabled or not. 

[Video](https://screenshot.click/17-32-qlayx-1zd15.mp4)

So maybe it can be a different selector: 
```css
.menu-drawer__utility-links:has(.menu-drawer__account + ul:empty) {
```
Something to test 🤔

Reply:
Damn, so yeah then maybe we need to rely on adding a class on the wrapper to know when locales are enable in the header.

Reply:
You're correct (I just edited my message to account for that actually haha), but it is _fairly_ bad support atm. If we think this is minor enough, that's fine, but probably still only is it really helps us avoid a much more complex fix.